src/cluster-api-provider-openshift-assisted.yaml

```diff
@@ -803,14 +812,74 @@ spec:
           capabilities:
             drop:
             - ALL
+        volumeMounts:
+        - mountPath: /tmp/k8s-webhook-server/serving-certs/
+          name: certs
       securityContext:
         runAsNonRoot: true
       serviceAccountName: capoa-bootstrap-controller-manager
       terminationGracePeriodSeconds: 10
+      volumes:
+      - name: certs
+        secret:
+          secretName: capoa-bootstrap-webhook-cert-secret
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: capoa-bootstrap-cert
+  namespace: capoa-bootstrap-system
+spec:
+  commonName: capoa-bootstrap-webhook-service.capoa-bootstrap-system.svc
+  dnsNames:
+  - capoa-bootstrap-webhook-service.capoa-bootstrap-system.svc
+  duration: 8760h
+  issuerRef:
+    kind: Issuer
+    name: capoa-bootstrap-selfsigned-issuer
+  renewBefore: 360h
+  secretName: capoa-bootstrap-webhook-cert-secret
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: capoa-bootstrap-selfsigned-issuer
+  namespace: capoa-bootstrap-system
+spec:
+  selfSigned: {}
+---
```

Comment:
I'm not sure how you end up having the cert-manager CRs into the src/cluster-api-provider-openshift-assisted.yaml file while you don't have it under the chart [templates](https://github.com/stolostron/cluster-api-installer/tree/main/charts/cluster-api-provider-openshift-assisted/templates), Is this mistake ?  We don't use cert-manager CRs we use openshift certificate service ?

Reply:
My understanding was that the `src` directory was just for reference and should reflect exactly what was pulled from upstream (before processing). So this all seems to be working that way.

Reply:
yes, you are right